vxquery-core/src/main/java/org/apache/vxquery/datamodel/values/ValueTag.java

```diff
@@ -82,6 +82,7 @@
     public static final int COMMENT_NODE_TAG = 105;
     public static final int PI_NODE_TAG = 106;
     public static final int NODE_TREE_TAG = 107;
+    public static final int OBJECT_TAG=109;
```

Comment:
Should this be JSON_OBJECT_TAG? I am wondering if its better to add JSON or leave it off?